src/test/java/org/gridsuite/filter/server/expertrule/NumberExpertRuleTest.java

```diff
@@ -442,35 +446,58 @@ private static Stream<Arguments> provideArgumentsForLoadTest() {
                 Arguments.of(GREATER_OR_EQUALS, FieldType.NOMINAL_VOLTAGE, 13.0, null, load, true),
                 Arguments.of(GREATER_OR_EQUALS, FieldType.NOMINAL_VOLTAGE, 14.0, null, load, false),
+                // Load fields
+                Arguments.of(GREATER_OR_EQUALS, FieldType.P0, 72.0, null, load, true),
+                Arguments.of(GREATER_OR_EQUALS, FieldType.Q0, 272.0, null, load, true),
 
                 // --- GREATER --- //
                 // VoltageLevel fields
                 Arguments.of(GREATER, FieldType.NOMINAL_VOLTAGE, 12.0, null, load, true),
                 Arguments.of(GREATER, FieldType.NOMINAL_VOLTAGE, 13.0, null, load, false),
                 Arguments.of(GREATER, FieldType.NOMINAL_VOLTAGE, 14.0, null, load, false),
+                // Load fields
+                Arguments.of(GREATER, FieldType.P0, 70.0, null, load, true),
+                Arguments.of(GREATER, FieldType.Q0, 270.0, null, load, true),
 
                 // --- LOWER_OR_EQUALS --- //
                 // VoltageLevel fields
                 Arguments.of(LOWER_OR_EQUALS, FieldType.NOMINAL_VOLTAGE, 14.0, null, load, true),
                 Arguments.of(LOWER_OR_EQUALS, FieldType.NOMINAL_VOLTAGE, 13.0, null, load, true),
                 Arguments.of(LOWER_OR_EQUALS, FieldType.NOMINAL_VOLTAGE, 12.0, null, load, false),
+                // Load fields
+                Arguments.of(LOWER_OR_EQUALS, FieldType.P0, 80.0, null, load, true),
+                Arguments.of(LOWER_OR_EQUALS, FieldType.Q0, 300.0, null, load, true),
+                Arguments.of(LOWER_OR_EQUALS, FieldType.Q0, 100.0, null, load, false),
 
                 // --- LOWER --- //
                 // VoltageLevel fields
                 Arguments.of(LOWER, FieldType.NOMINAL_VOLTAGE, 14.0, null, load, true),
                 Arguments.of(LOWER, FieldType.NOMINAL_VOLTAGE, 13.0, null, load, false),
                 Arguments.of(LOWER, FieldType.NOMINAL_VOLTAGE, 12.0, null, load, false),
+                // Load fields
+                Arguments.of(LOWER, FieldType.P0, 99.0, null, load, true),
+                Arguments.of(LOWER, FieldType.Q0, 300.0, null, load, true),
 
                 // --- BETWEEN --- //
                 // VoltageLevel fields
                 Arguments.of(BETWEEN, FieldType.NOMINAL_VOLTAGE, null, Set.of(12.0, 14.0), load, true),
                 Arguments.of(BETWEEN, FieldType.NOMINAL_VOLTAGE, null, Set.of(13.5, 14.0), load, false),
+                // Load fields
+                Arguments.of(BETWEEN, FieldType.P0, null, Set.of(60.0, 80.0), load, true),
+                Arguments.of(BETWEEN, FieldType.P0, null, Set.of(60.0, 70.0), load, false),
+                Arguments.of(BETWEEN, FieldType.Q0, null, Set.of(270.0, 280.0), load, true),
+                Arguments.of(BETWEEN, FieldType.Q0, null, Set.of(100.0, 260.0), load, false),
 
                 // --- EXISTS --- //
```

Comment:
Lack of tests for EXISTS, note that at the moment we use functional test approach, so need to cover all operators for each field of each equipment 

EXIST for number type is defined at : https://github.com/gridsuite/gridexplore-app/blob/main/src/components/dialogs/filter/expert/expert-filter-utils.ts#L65C1-L73C41

Reply:
Done.